framework/src/org/checkerframework/framework/type/AnnotationClassLoader.java

```diff
@@ -727,20 +709,26 @@ public int compare(File o1, File o2) {
      *
      * @param fullyQualifiedAnnoNames a set of strings where each string is a single annotation
      *     class's fully qualified name
+     * @param issueWarnings set to true to issue a warning when a loaded annotation is not a type
+     *     annotation. It is useful to set this to true if the given set of annotations must all be
+     *     well-defined type annotations (eg annotation class names given as command line
+     *     arguments). This should be set to false if the given set of annotations contain
+     *     meta-annotations or non-type annotations.
      * @return a set of loaded annotation classes
-     * @see #loadAnnotationClass(String)
+     * @see #loadAnnotationClass(String, boolean)
      */
-    private final Set<Class<? extends Annotation>> loadAnnotationClasses(
-            final /*@Nullable*/ Set<String> fullyQualifiedAnnoNames) {
+    protected final Set<Class<? extends Annotation>> loadAnnotationClasses(
+            final /*@Nullable*/ Set<String> fullyQualifiedAnnoNames, boolean issueWarnings) {
         Set<Class<? extends Annotation>> loadedClasses =
                 new LinkedHashSet<Class<? extends Annotation>>();
 
         if (fullyQualifiedAnnoNames != null && !fullyQualifiedAnnoNames.isEmpty()) {
             // loop through each class name & load the class
             for (String fullyQualifiedAnnoName : fullyQualifiedAnnoNames) {
-                Class<? extends Annotation> annoClass = loadAnnotationClass(fullyQualifiedAnnoName);
-                if (annoClass != null) {
-                    loadedClasses.add(annoClass);
+                Class<? extends Annotation> anno =
```

Comment:
Why not keep the `annoClass` name? Other local variables of this type have that name.

Reply:
Fixed